pkg/flare/config_check.go

```diff
@@ -144,5 +144,21 @@ func PrintConfig(w io.Writer, c integration.Config) {
 		state := fmt.Sprintf("dispatched to %s", c.NodeName)
 		fmt.Fprintln(w, fmt.Sprintf("%s: %s", color.BlueString("State"), color.CyanString(state)))
 	}
+	printReportingDisabledInfo(w, c)
```

Comment:
nit: This only needs to be called when `len(c.ADIdentifiers) > 0`.
Perhaps we should move it up in the `if len(c.ADIdentifiers) > 0 {...` branch? configs without `ADIdentifiers` can never match a container exclusion rule.

Reply:
Done 👍